sample_projects/LEQuad/lequad.hpp

```diff
@@ -223,18 +222,19 @@ class LEQuad
     ahrs_t ahrs;                                                ///< The attitude estimation structure
     Ahrs_ekf ahrs_ekf;
 
+    Position_estimation position_estimation;                    ///< The position estimaton structure
```

Comment:
No you cannot create an INS (abstract) :-) 
I let Position_estimation because it was tested in flight. We will be able to choose between complementary and kalman once @Spyroth is done fixing and testing INS_kf :+1: